src/app/core/models/part-model.ts

```diff
@@ -1,9 +1,18 @@
 export class PartModel {
     static create(params: { [field: string]: any }): PartModel {
-        return new PartModel(params.id, params.title, params.description);
+        if (!params.properties) {
+            params.properties = [];
+        }
+        return new PartModel(params.id, 
```

Comment:
please consider to pass params object not 5 arguments

Reply:
constructor( params: Partial<IParams>,
public id: number = params.id,
                public title: string =,
                public description: string) {}
                public properties: string[],
                public description: string,
                public owner: string) {}